src/pytest_ansible/plugin.py

```diff
@@ -175,6 +201,18 @@ def pytest_configure(config):
         inject(start_path)
 
 
+def pytest_collect_file(
+    file_path: Path | None,
+    parent: pytest.Collector,
+) -> Node | None:
+    """Transform each found molecule.yml into a pytest test."""
```

Comment:
here, something like:

```
if not HAS_MOLECULE:
 return None
 
```

this will prevent the import error if molecule is not installed, but we will need to install it for the tests to run in CI

Reply:
yes the molecule is installed but still its giving this ModuleNotFoundError in CI